src/renderer/modules/webpack/get-modules.ts

```diff
@@ -0,0 +1,182 @@
+import type { Filter, GetModuleOptions, RawModule } from "src/types";
+import { wpRequire } from "./patch-load";
+import { logError } from "./util";
+
+// Export-finding utilities
+
+/**
+ * Get the default export (or top-level exports) for a module
+ * @param m Raw module
+ * @returns Exports
+ * @internal
+ */
+export function getExports<T>(m: RawModule): T | undefined {
+  if (typeof m.exports === "object" && m.exports) {
+    if (Object.keys(m.exports).length === 1) {
+      for (const key of ["Z", "default"] as const) {
```

Comment:
Where is `ZP`?

Reply:
At the time we set these, during the original rewrite, `Z` was the most common default export key for modules built with SWC, and `default` was for stragglers that had been compiled with Babel. Is `ZP` common enough at this point to add it to this list? Currently I see:

- 13 modules with ZP as their sole export
- 154 modules with ZP as an export in general, including other top-level exports
- 1364 modules with Z as their sole export
- 1547 modules with Z as an export in general

Reply:
I think it is important enough. I have only seen Z and ZP as sole export names. Given that ZP exists as a sole export at all makes it important enough in my eyes.